### DISCUSSION TITLE
FIltering project per user

### DISCUSSION THREAD

**Discussion by fturibak** [ludeknovy/jtl-reporter#192] [General]
Hello!

Me again :-) I think you will start to hate me :P

I've created 2 users **user1** and **user2** with roles operator. Is there a possibility that **user1**  sees only projects assigned to him, not all projects.
Currently both users sees all projects what's not ideal for some kind of shared production environment.

Some kind of multi-tenancy would be nice to have feature.

Thanks

**Comment by ludeknovy** [comment]
Hi @fturibak !

This makes total sense, I will need to investigate this first.

**Reply by ludeknovy** [reply]
@fturibak I have it almost ready?
1. Admin role has always access to all projects
2. Operator role has access only to projects that it created and projects with granted access (by admin role)

**Reply by fturibak** [reply]
Great news!

Maybe **readonly** role should also have restricted access to projects granted by admin...

**Reply by ludeknovy** [reply]
Ahh, sure, I forgot to mention it, but that's currently the case.

**Reply by ludeknovy** [reply]
@fturibak merged the changes. They are available in the "latest"  images.

**Reply by ludeknovy** [reply]
@fturibak did you have a chance to check it out?

**Reply by fturibak** [reply]
Hi @ludeknovy ! 

I'm not longer on the project where I've been using JTLReporter. 

If memory serves me right this was behaving as expected.

**Reply by ludeknovy** [reply]
Ah, sorry to hear that. Anyway, thanks for the confirmation and good luck with your new project!
